docs/_sass/_elements.scss

```diff
@@ -152,14 +152,15 @@
 // ***********************
 
 .hover-button {
-	float: right;
+	position: absolute;
 	width: 30px;
 	height: 30px;
 	background: url("../../img/copy-to-clipboard.svg");
 	background-repeat: no-repeat;
 	background-position: center;
 	border: none;
 	cursor: pointer;
+  right: 24px;
```

Comment:
It looks like this file is using tabs for indentation, and that you've used spaces here (I see the same pattern below, too). Can this be updated to be more consistent?